src/chainparams.cpp

```diff
@@ -96,26 +97,30 @@ const CChainParams &Params() {
     return *globalChainParams;
 }
 
-std::unique_ptr<const CChainParams> CreateChainParams(const ArgsManager& args, const std::string& chain)
+std::unique_ptr<const CChainParams> CreateChainParams(const ArgsManager& args, const ChainType& chain)
```

Comment:
In commit "refactor: Replace string chain name constants with ChainTypes" (5589a4b0dd5b25017299f666127b7537f5f7a4c1)

Maybe replace `const ChainType&` with `ChainType` everywhere since it's an enum and probably more efficient to pass by value instead of by reference